lib/rules/annotation-no-unknown/README.md

```diff
@@ -37,12 +43,21 @@ a {
 
 ## Optional secondary options
 
-### `ignoreAnnotations: ["/regex/", /regex/, "string"]`
+### `ignoreAnnotations`
```

Comment:
[note] This change breaks a permalink with `#`, but I believe it's worth defining an option's schema in JSON, instead of a heading text.

Reply:
SGTM.